posthog/models/event/sql.py

```diff
@@ -14,6 +23,8 @@
 )
 DROP_EVENTS_TABLE_SQL = lambda: f"DROP TABLE IF EXISTS {EVENTS_DATA_TABLE()} ON CLUSTER '{settings.CLICKHOUSE_CLUSTER}'"
 
+INSERTED_AT_COLUMN = ", inserted_at DateTime64(6, 'UTC') NULL DEFAULT NULL"
```

Comment:
What's the difference between NULL and Nullable btw?

Reply:
It's the same, but for some reason `ADD COLUMN` wasn't working with `NULL DEFAULT NULL` so I swapped that for `Nullable`.

Reply:
Can use `Nullable` in both just for clarity...

Reply:
Latest commit is using `Nullable` in both to avoid confusion (had to rebase anyways to fix tests). `Nullable` works the same as `NULL` in the sense that you can store `NULL` alongside values of `DateTime64(6, 'UTC')`. They do differ in that `Nullable` also sets the default to `NULL` but I'm doing that explicitly too.